python/ray/air/tests/test_dataset_config.py

```diff
@@ -283,9 +283,9 @@ def checker(shard, results):
         assert results[0] != results[1], results
         stats = shard.stats()
         assert (
-            "Stage 1 read->randomize_block_order->map_batches: 1/1 blocks executed "
-            in stats
-        ), stats
+            "Stage 1 read->randomize_block_order->"
+            "transform_batch: 1/1 blocks executed "
+        ) in stats, stats
```

Comment:
nit: should `in stats` be within the parentheses here?

Reply:
In this case the parentheses are being used to concatenate the string which doesn't fit on one line. Does

```python
assert (
    "Stage 1 read->randomize_block_order->"
    "transform_batch: 1/1 blocks executed "
in stats), stats
```

do the same thing here? This is a really unusual case where I actually don't know what python does - does it concatenate the string before evaluating `<string> in stats`?

After a bit of testing it seems like you can do it either way. To me it makes sense to group the string together first, like this:

```python
assert (<multiline string>) in stats, stats
```

though if you'd like I can change it.

Reply:
Yeah, I think either way works, I was just judging based off the existing test case. The existing test case checked `assert (string in stats)`, with the parentheses containing the expression that evaluates to a boolean value. In the case of `assert (string) in stats`, I think this also accomplishes the same thing. Feel free to follow your preference

Reply:
Wow, somehow I didn't spot this the first time around. Thanks for pointing this out, I think I'll change it back for consistency!